.gitignore

```diff
@@ -8,3 +8,5 @@ build/*
 .idea
 *.iml
 
+
```

Comment:
add a comment indicating that this is a directory of releases 

```
# Releases

```

Reply:
done